bounties/npm/git-commit-range/1/vulnerability.json

```diff
@@ -0,0 +1,51 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2020-06-30",
+  "AffectedVersionRange": "*",
+  "Summary": "RCE on \"git-commit-range\"",
+  "Author": {
+    "Username": "Mik317",
+    "Name": "Michele Romano"
+  },
+  "Package": {
+    "Registry": "npm",
+    "Name": "git-commit-range",
+    "URL": "https://www.npmjs.com/package/git-commit-range"
+  },
+  "CWEs": [
+    {
+      "ID": "78",
+      "Description": "OS command injection"
+    }
+  ],
+  "CVSS": {
+    "Version": "3.1",
+    "AV": "",
+    "AC": "",
+    "PR": "",
+    "UI": "",
+    "S": "",
+    "C": "",
+    "I": "",
+    "A": "",
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/aichbauer/node-git-commit-range",
+    "Codebase": [
+      "NodeJS"
+    ]
+  },
+  "Permalinks": [
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L32",
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L44 ",
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L46 ",
+    " https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L50",
```

Comment:
Remove space at the beginning of the string

Reply:
Done